deploy/helm/sumologic/templates/chart-configmap.yaml

```diff
@@ -3,6 +3,6 @@ kind: ConfigMap
 metadata:
   name: sumologic-configmap
 data:
-  fluentdLogs: {{ template "sumologic.metadata.name.logs.service" . }}
+  otelcolLogs: {{ template "sumologic.metadata.name.logs.service" . }}
```

Comment:
could we name it `logs`, `metadataLogs` or `logsMetadata`?

I always felt that using specific component name was bad decision

Reply:
yeah, maybe I'll go for `metadataLogs`